tools/src/versioning/android/versionCxx/patches/expo-modules-core.patch

```diff
@@ -198,11 +202,24 @@ index ebd5c60547..c469341691 100644
    static auto constexpr TAG = "JSIInteropModuleRegistry";
  
    static jni::local_ref<jhybriddata> initHybrid(jni::alias_ref<jhybridobject> jThis);
+diff --git a/packages/expo-modules-core/android/src/main/cpp/JavaCallback.h b/packages/expo-modules-core/android/src/main/cpp/JavaCallback.h
+index 22453f4a08..febc000b0c 100644
+--- a/packages/expo-modules-core/android/src/main/cpp/JavaCallback.h
++++ b/packages/expo-modules-core/android/src/main/cpp/JavaCallback.h
+@@ -15,7 +15,7 @@ namespace expo {
+ class JavaCallback : public jni::HybridClass<JavaCallback> {
+ public:
+   static auto constexpr
+-    kJavaDescriptor = "Lexpo/modules/kotlin/jni/JavaCallback;";
++    kJavaDescriptor = "L{VERSIONED_ABI_NAME}/expo/modules/kotlin/jni/JavaCallback;";
```

Comment:
It seems to be very annoying to update that in each cycle. Maybe it would be better to add that to the package code. So we would have a `PACKAGE_PREFIX` variable declared in the CMake and use that to prefix all strings like this. It obviously would be an empty string in a typical use case, but we could change that during the versioning process. What do you think about that? It's not for now.

Reply:
it's also be annoying to me. i'm thinking to have a heuristic transform first and do patching then. will update here of how's it going for the heuristic approach.